commons/src/types/Nylas.ts

```diff
@@ -225,6 +225,7 @@ export interface Thread {
   unread: boolean;
   version: number;
   expanded?: boolean;
+  folders?: string[];
```

Comment:
I know won't be using this right now, but the proper signature for this should be updated as follows:

```
export interface Folder {
   object?: "folder";
   display_name: string;
   id: string;
   name: string;
}
```

Then instead of `string[]` it will be:
```
folders?: Folder[];
```

I took the types from https://developer.nylas.com/docs/api/#get/threads

Reply:
thanks!